services/brig/test/integration/API/User/Auth.hs

```diff
@@ -439,6 +442,57 @@ testNewSessionCookie config b = do
         const 200     === statusCode
         const Nothing === getHeader "Set-Cookie"
 
+testSuspendInactiveUsers :: HasCallStack => Maybe Opts.Opts -> Brig -> Http ()
```

Comment:
question for the reviewer: this is currently taking >30s, and i'm not sure it'll work reliably on different machines and timing contexts.  is there anything we could do about this?

(for once, i'm tempted to change the config back from 29s to maybe 15s, and if concourse makes it pass twice we'll merge it and hope for the best.)